modular_zubbers/code/modules/antagonists/changeling/changeling.dm

```diff
@@ -0,0 +1,3 @@
+/datum/antagonist/changeling
+	genetic_points = 10
+	total_genetic_points = 10
```

Comment:
I'd rather just comment out the skyrat changes
//genetic points = whatever //BUBBER EDIT REMOVAL
Yes this will add more merge conflicts but it'll also make it so there's less code to deal with.